### PR TITLE
install-conda-dependencies.sh: do not modify .bashrc

### LINKED ISSUE #24: install-conda-dependencies modifies .bashrc without asking the user

### PR DESCRIPTION
**Pull request by matthiasdiener**
Fixes #24.

### CODE REVIEW

File: install-conda-dependencies.sh [hunk 1]
```diff
@@ -7,4 +7,6 @@ echo "==== Installing Conda packages"
 conda install --yes pocl numpy pyvisfile pyopencl
 
 export OCL_ICD_VENDORS=$MY_CONDA_DIR/etc/OpenCL/vendors/
-echo 'export OCL_ICD_VENDORS=$CONDA_PREFIX/etc/OpenCL/vendors/' >> $HOME/.bashrc
+
+echo "To use the just-installed pocl, add the following line to your ~/.bashrc "
+echo 'export OCL_ICD_VENDORS=$CONDA_PREFIX/etc/OpenCL/vendors/'
```
Comment (inducer) [inline install-conda-dependencies.sh:12]: Why is this necessary? `ocl-icd` as installed via conda should default to this.

Comment (matthiasdiener) [reply install-conda-dependencies.sh]: Fixed.
